src/gsy_e/models/market/two_sided.py

```diff
@@ -429,3 +429,8 @@ def replace_recommendations_with_residuals(recommendation: Dict):
             recommendations = [replace_recommendations_with_residuals(recommendation)
                                for recommendation in recommendations]
         return recommendations
+
+    @staticmethod
+    def representation_name():
+        """Return the market type representation."""
+        return "Spot Market"
```

Comment:
I think this is not needed, since `TwoSidedMarket` inherits from `OneSidedMarket`.

Reply:
It indeed does, I have removed from here